kolibri_explore_plugin/assets/src/components/DiscoveryNavBar.vue

```diff
@@ -58,6 +60,9 @@
       logo() {
         return assets.EndlessLogo;
       },
+      showDiscoveryTab() {
+        return !plugin_data.hideDiscoveryTab;
+      },
```

Comment:
It seems to me that this method is not used anywhere? Apologies if this is unhelpful.

Reply:
It is called in ` <b-nav-text v-if="showDiscoveryTab"`, which in my understanding means that the whole tag will only be rendered when that method returns true.

I'm also completely new to Vue, but this is what I'm basing my understanding on: https://vuejs.org/guide/essentials/conditional.html